README.md

```diff
@@ -245,3 +268,4 @@ status:
 ## Links
```

Comment:
I'm not 100% sure what this section is for, but if the gain access link is listed here, shouldn't the new `Splunk & app-sre` link be here too?

Reply:
Indeed. This PR was started before the doc. improvement so the links section did not exist yet.

Reply:
Elaborating:  The idea with the links section was avoiding rewrite stuff that is written elsewhere - or with a deeper coverage. The way I linked the `Getting access` and the `Splunk & app-sre` was done before the links section so the reason they were not in that section - but now they are.